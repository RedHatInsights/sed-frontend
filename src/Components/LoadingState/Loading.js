import React from 'react';
import { Bullseye, Spinner } from '@patternfly/react-core';

const Loading = () => {
  return (
    <Bullseye>
      <Spinner />
    </Bullseye>
  );
};

export default Loading;
