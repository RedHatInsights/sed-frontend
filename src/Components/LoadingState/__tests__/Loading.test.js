import React from 'react';
import { render } from '@testing-library/react';
import Loading from '../Loading';

it('renders correctly', () => {
  const { container } = render(<Loading />);
  expect(container).toMatchSnapshot();
});
