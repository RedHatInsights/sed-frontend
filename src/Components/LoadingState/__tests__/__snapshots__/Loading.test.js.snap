// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="pf-v5-l-bullseye"
  >
    <svg
      aria-label="Contents"
      aria-valuetext="Loading..."
      class="pf-v5-c-spinner pf-m-xl"
      role="progressbar"
      viewBox="0 0 100 100"
    >
      <circle
        class="pf-v5-c-spinner__path"
        cx="50"
        cy="50"
        fill="none"
        r="45"
      />
    </svg>
  </div>
</div>
`;
